audio_video_recorder/README.md

```diff
@@ -0,0 +1,78 @@
+# audio_video_recorder
+
+ROS package for recording audio and video synchronously
+
+![[Full video on Google Drive](https://drive.google.com/file/d/1TWnRKbOdq6jPza82eNhhjn56lQXRxWjl/view?usp=sharing)
+](.media/pr2_sample.gif)
+
+[Full video on Google Drive](https://drive.google.com/file/d/1TWnRKbOdq6jPza82eNhhjn56lQXRxWjl/view?usp=sharing)
+
+## Sample
+
+You can record audio and video on your laptop.
+
+```bash
+roslaunch audio_video_recorder sample_audio_video_recorder.launch
+```
+
+## Parameters
+
+Node: `audio_video_recorder/audio_video_recorder`
+
+### Common parameters
+
+- `queue_size` (`Int`, default: `100`)
+
+  Queue size
+
+- `do_timestamp` (`Bool`, default: `true`)
+
+  Do timestamp or not (`false` is not working now.)
+
+- `file_location` (`String`, default: `/tmp/test.avi`)
+
+  Output file location
+
+- `file_format` (`String`, default: `avi`)
+
+  Output file format (Only `avi` is supported now.)
+
+### Audio parameters
+
+- `audio_format` (`String`, default: `mp3`)
+
+  Audio format
```

Comment:
what is other option?

Reply:
Other option is `mp3` or `wave`.
I will add to readme.